build_defs/python.build_defs

```diff
@@ -0,0 +1,72 @@
+subinclude("///proto//build_defs/sdk", "///python//build_defs:python")
```

Comment:
These subincludes might not work if they define the path differently in the host repo. I think they might need to be configuration options. This feels like a bit of a shortcoming of the plugins API though.

Reply:
Yeah, I made the assumption for now given the current API. We should look into it.